docker-compose.yml

```diff
@@ -65,18 +76,24 @@ services:
       memlock:
         soft: -1
         hard: -1
-    network_mode: host
+    ports:
+      - "9200:9200"
+    networks:
+      - protectedplanet
     env_file:
       - '.env'
     volumes:
       - 'protectedplanet_es_data:/usr/share/elasticsearch/data'
   kibana:
     image: docker.elastic.co/kibana/kibana:8.6.0
     environment:
-      - ELASTICSEARCH_HOSTS=http://localhost:9200
+      - ELASTICSEARCH_HOSTS=http://protectedplanet-elasticsearch:9200
     env_file:
       - '.env'
-    network_mode: host
+    ports:
+      - "5601:5601"
```

Comment:
Please change to "5601" (without host port), so it won't conflict with host port. This is Jenkins test, no one will connect from the host.